### PR TITLE
Refine the command logic and descriptions

### PR DESCRIPTION
**Pull request by ShichaoQiu**
Refine the command logic and descriptions

### CODE REVIEW

File: src/spring-cloud/azext_spring_cloud/custom.py [hunk 1]
```diff
@@ -921,11 +932,40 @@ def _app_deploy(client, resource_group, service, app, name, version, path, runti
         source=user_source_info)
 
     # upload file
+    logger.warning("[2/3] Uploading package to blob")
     file_service = FileService(storage_name, sas_token=sas_token)
     file_service.create_file_from_path(share_name, None, relative_path, path)
-    logger.info("Upload file succeed, start to update deployment.")
+
+    if file_type == "Source" and ~no_wait:
```
[the remark this answers is not in the document]
Comment (MyronFanQiu) [reply src/spring-cloud/azext_spring_cloud/custom.py]: Usually, `no_wait` should be `None` or `True`

File: src/spring-cloud/azext_spring_cloud/custom.py [hunk 2]
```diff
@@ -921,11 +932,40 @@ def _app_deploy(client, resource_group, service, app, name, version, path, runti
...
+                    resource_group_name=resource_group,
+                    service_name=service,
+                    app_name=app,
+                    deployment_name=name).url
+                return log_url
+            except CloudError:
+                return None
+
+        def get_logs_loop():
+            old_log_url = get_log_url()
```
Comment (MyronFanQiu) [inline src/spring-cloud/azext_spring_cloud/custom.py]: Looks like the `log_url` will change once. Just wonder, what if this `old_log_url` has changed when you call it first time? Is it possible?

File: src/spring-cloud/azext_spring_cloud/custom.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (ShichaoQiu) [reply src/spring-cloud/azext_spring_cloud/custom.py]: Good suggestions, have updated.

File: src/spring-cloud/azext_spring_cloud/custom.py (hunk 2 above)
Comment (ShichaoQiu) [reply src/spring-cloud/azext_spring_cloud/custom.py]: Good questions, we have refined the logic to ensure that `old_log_url` will always be the real old value. In the old logic, when we call it first time, `old_log_url` may be the `new value` sometimes.

**Pull request merged**